Tests/AshtonTests.swift

```diff
@@ -163,6 +163,16 @@ class AshtonTests: XCTestCase {
         }
     }
 
+    func testUnknownFontParsing() {
```

Comment:
Test looks good. Can we add a test where we check the opposite - that no font is reported. Maybe we can check this for all available system fonts.